src/test/java/uk/gov/hmcts/reform/unspec/sampledata/CaseDataBuilder.java

```diff
@@ -208,6 +205,88 @@ public CaseDataBuilder legacyCaseReference(String legacyCaseReference) {
         return this;
     }
 
+    public CaseDataBuilder atState(FlowState.Main flowState) {
```

Comment:
should we raise refactor ticket for updating all classes to use this method and make all specific ones private?